service.partymode.autostart/main.py

```diff
@@ -0,0 +1,20 @@
+import xbmc, xbmcaddon
+
+def handleArg(arg):
```

Comment:
Unused

Reply:
Removed...but for me could be stay there. Maybe, could be useful for some new features